.gitignore

```diff
@@ -22,3 +22,4 @@ MANIFEST
 .env
 .venv
 venv/
+venv2/
```

Comment:
why do you need a `venv2/`?

Reply:
`venv` ignoring should go in your global gitignore

Reply:
nah i think it's fine, but definitely not a `venv2`

Reply:
naming it `venv` is def a dev choice though, other devs shouldn't have to name their environments the same as yours or have to add them all to ignore